### PR TITLE
updated docs

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] Ready

## Related Issues
relates: https://jira-dc.paloaltonetworks.com/browse/XSUP-30281

## Description
updating the docker image and the REAMME of the command ***msgraph-get-managed-device-by-id*** that by default physicalMemoryInBytes will return 0 from the command.

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@eyalpalo Doc review completed.

File: Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ After you successfully execute a command, a DBot message appears in the War Room
 ### msgraph-get-managed-device-by-id
 ***
 Get managed devices
+ - Please notice that by default physicalMemoryInBytes will return 0 since Microsoft haven't updated the v1.0 schema with that information yet.
```
Comment (eyalpalo) [inline Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/README.md]:
```suggestion
 -Note: physicalMemoryInBytes will return 0 by default as Microsoft v1.0 schema is not updated with that information.
```

File: Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Microsoft Endpoint Manager (Intune)
+
+- Added a note in the README file that by default physicalMemoryInBytes will return 0 from the command ***msgraph-get-managed-device-by-id***,
```
Comment (eyalpalo) [inline Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md:6]: The README is not relevant to the RN. 
We should:
1. Add the note from the readme also to the yml file.
2. Update the RN accordingly.

**Review by eyalpalo — CHANGES_REQUESTED** [review]
nice
remove the package-lock.json from the PR pls.

File: Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md (hunk 2 above)
Comment (maimorag) [reply Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md]: done

File: Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Endpoint Manager (Intune)
+- Updated the Docker image to: *demisto/crypto:1.0.0.82686*.
+- Updated the yml file in ***msgraph-get-managed-device-by-id*** command's description, that physicalMemoryInBytes will return 0 by default.
```
Comment (eyalpalo) [inline Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md:6]:
```suggestion
- Added a note regarding ***msgraph-get-managed-device-by-id*** command, that physicalMemoryInBytes is 0 by default.
```

**Review by eyalpalo — APPROVED** [review]

**Pull request merged**